src/main/webapp/i18n/de/editor.json

```diff
@@ -6,17 +6,26 @@
             },
             "title": "Code Editor für Übung:",
             "instructions": "Aufgabenstellung &nbsp;",
-            "unsavedChanges": "Nicht gespeicherte Änderungen vorhanden.",
+            "unsavedChanges": "Ungespeichert.",
+            "unsavedChangesTooltip": "Nicht gespeicherte Änderungen vorhanden.",
             "savingChanges": "Änderungen werden gespeichert.",
+            "savingChangesTooltip": "Speichert...",
```

Comment:
Shouldn't the text for savingChanges and savingChangesTooltip be switched? :)

Reply:
Correct :)